### PR TITLE
Add couchdb.cr to the list

### PR DESCRIPTION
**Pull request by Schniz**

### CODE REVIEW

**Comment by veelenga** [comment]
@Schniz @TechMagister looks like it is not maintained. This shard doesn't compile

File: README.md [hunk 1]
```diff
@@ -193,6 +193,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## Database Drivers/Clients
  * [arangocr](https://github.com/solisoft/arangocr) - ArangoDB client
+ * [couchdb.cr](https://github.com/TechMagister/couchdb.cr) - CouchDB client
```
Comment (veelenga) [inline README.md]: @Schniz @TechMagister looks like it is not maintained. This shard doesn't even compile

**Comment by Schniz** [comment]
Well, it was just reviewed and merged 😸 
Everything compiles on my machine, and tests pass.

**Comment by veelenga** [comment]
Yes, it now works for me. However, to be able to merge I would like to see at least one published release and CI builds. See official guidelines on how to create/maintain a shard: https://crystal-lang.org/docs/guides/writing_shards.html

Also, we are thinking about required weekly CI builds (#347) for new entries. For a moment, it is not a blocker for you, but if you want, you can experiment.

Making the list awesome and up to date is not that easy. We want to add the best shards and be sure they are propertly maintained. Thanks for understanding.

**Comment by Schniz** [comment]
Hi, the first version was just released: https://github.com/TechMagister/couchdb.cr/releases/tag/v0.1

> Also, we are thinking about required weekly CI builds...

Sounds awesome and very reasonable. I'll try making a CI config for this project too - so it won't break over time.

> Making the list awesome and up to date is not that easy

I'm sure - and I'm thankful that you're doing that. Thanks ✨

**Comment by Schniz** [comment]
Hi! Now it contains a CI badge

**Pull request merged**